packages/components/psammead-most-read/src/title.jsx

```diff
@@ -0,0 +1,25 @@
+import React from 'react';
+import styled from 'styled-components';
+import { shape, string } from 'prop-types';
+import { C_SHADOW } from '@bbc/psammead-styles/colours';
+import { getTrafalgar } from '@bbc/gel-foundations/typography';
+import { scriptPropType } from '@bbc/gel-foundations/prop-types';
+import { getSansRegular } from '@bbc/psammead-styles/font-styles';
+
+const MostReadTitleWrapper = styled.h2`
```

Comment:
It'd be better to name styled components related to what the underlying HTML element is. e.g.
```suggestion
const StyledHeading = styled.h2`
```
This is in keeping with the naming convention in other Psammead packages.